VTEX - GiftCard Hub API.json

```diff
@@ -2434,56 +2502,60 @@
                         "$ref": "#/components/schemas/Shipping"
                     }
                 },
-                "example": {
-                    "orderId": "v2915869org-01",
-                    "sequence": 2915869,
+                "default": {
```

Comment:
```suggestion
                "example": {
```

Reply:
Only use `default` for fields that:
- Are not required.
- Are in request the request body.
- Have a default value that influences the platform behavior if you don't send a custom one in the request.